package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aframe-resonance-audio-component",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
revert to version `0.1.0` so that after merge could increment version when publishing new release